openedx/core/djangoapps/xmodule_django/models.py

```diff
@@ -181,6 +183,19 @@ class UsageKeyField(OpaqueKeyField):
     KEY_CLASS = UsageKey
 
 
+class UsageKeyWithRunField(UsageKeyField):
```

Comment:
For folks with Open EDX installs that still use the XML Modulestore, do block structures just not work for them?